6.2_2000_run.py

```diff
@@ -15,6 +15,7 @@ def timer(f, *args, **kwargs):
     print("Execution time: {:.6f} seconds".format(end_time - start_time))
 
 
-timer(print, "Hello")
-timer(zip, [1, 2, 3], [4, 5, 6])
-timer("Hi {name}".format, name="Bug")
+if __name__ == '__main__':
+    timer(print, "Hello")
```

Comment:
This is still not exactly perfect.


Your code should have a main function, proving you managed to meet the exercise demands.
every exercise should end with:
If __name__ == '__main__':
    main()

see more here:
https://realpython.com/python-main-function/#a-basic-python-main